src/documents/forms/EditDocumentForm.js

```diff
@@ -80,19 +91,25 @@ const EditDocumentForm = React.forwardRef(
                 <Form.Field required>
```

Comment:
could these be converted to use the `/forms/FormField` component to start to standardize inputs across the application?